### PR TITLE
added events, moved things

### PR DESCRIPTION
**Pull request by liuggio**
- renamed usecase in steps
- moved the doctrine things into the infrastructure folder
- create bot as service in the container for the command
- some fix to the steps...
- get random package works...

### CODE REVIEW

File: src/Infrastructure/Bundle/Tests/Controller/DefaultControllerTest.php [hunk 1]
```diff
@@ -8,6 +8,8 @@ class DefaultControllerTest extends WebTestCase
 {
     public function testIndex()
     {
+        $this->markTestSkipped();
+
         $client = static::createClient();
 
         $crawler = $client->request('GET', '/hello/Fabien');
```
Comment (giorrrgio) [inline src/Infrastructure/Bundle/Tests/Controller/DefaultControllerTest.php]: Shouldn't we remove the DefaultControllerTest?

File: src/Package/PackageRepository.php [hunk 2]
```diff
@@ -12,8 +14,21 @@ function __construct(ProviderInterface $provider)
...
+    public function getPackage($repository)
     {
-        return $this->provider->getANeverVisitedPackage();
+        $repo = $this->provider->get($repository);
+
+        if (!$repo) {
+            throw new \Exception('Repository name not valid');
+        }
+
+        $repo = Package::createFromPackage($repo);
```
Comment (giorrrgio) [inline src/Package/PackageRepository.php]: $repo = Package::createFromPackage($repo);

something wrong with naming?

File: src/Infrastructure/Bundle/Tests/Controller/DefaultControllerTest.php (hunk 1 above)
Comment (liuggio) [inline src/Infrastructure/Bundle/Tests/Controller/DefaultControllerTest.php]: :+1:

File: src/Package/PackageRepository.php (hunk 2 above)
Comment (liuggio) [inline src/Package/PackageRepository.php]: ops yes a lot of problems :)
WDYT `$package = Package::createFromAPI($repo);`

File: src/Infrastructure/Bundle/Tests/Controller/DefaultControllerTest.php (hunk 1 above)
Comment (marco-loche) [inline src/Infrastructure/Bundle/Tests/Controller/DefaultControllerTest.php]: Why don't remove the DefaultController at all?

**Pull request merged**